src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,60 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+import duke.taskList.TaskList;
+
+
+/**
+ * deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+
+    /**
+     * The constructor of the Storage class.
+     */
+    public Storage() {
+
+    }
+
+    /**
+     * Load the data from duke.txt into the TaskList inside of the program.
+     * @param list the TaskList that the data is loaded to from duke.txt.
+     */
+    public void load(TaskList list) {
+        try {
+            File theDir = new File("./src/main/data");
+            if (!theDir.exists()) {
+                theDir.mkdirs();
+            }
+            File myObj = new File("./src/main/data/duke.txt");
+            Scanner myReader = new Scanner(myObj);
+            while (myReader.hasNextLine()) {
+                String data = myReader.nextLine();
+                String type = data.substring(0, 1);
+                switch (type) {
```

Comment:
The `switch-case` statement needs to follow the coding standard stated in the course (no indentation for the case clause).